src/app/layout.tsx

```diff
@@ -13,7 +14,10 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body>{children}</body>
+      <body>
+        <Navbar />
+        <div className="pt-16">{children}</div>
```

Comment:
should be fine to remove the `pt-16`, the Hero should handle the spacing correctly I think